### ISSUE TITLE
Breaks forward privacy

### ISSUE THREAD

**Issue by jech** [pion/webrtc-zero-downtime-restart#2]
If I read this code correctly, it is persisting the whole `dtls.State`.  If I understand this correctly, the it is persistig the session key, which breaks forward secrecy.

Would it not be better to just persist the certificate, and renegotiate at reconnect?  This would cost just one round-trip, without breaking forward secrecy.

At any rate, I recommend having this idea reviewed by a competent cryptographer.

**Comment by Sean-Der** [comment]
@jech I agree, I don't think this is a good solution! I am going to update the README. 

* Complexity of implementing it
* Privacy+Security
* Handling across versions
